elixir/apps/domain/priv/repo/manual_migrations/20250725031509_reindex_flows_on_expires_at.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Domain.Repo.Migrations.IndexFlowsOnExpiresAt do
+  use Ecto.Migration
+
+  @disable_ddl_transaction true
+
+  def change do
+    drop_if_exists(index(:flows, [:account_id, :expires_at, :gateway_id], concurrently: true))
```

Comment:
I think I remember dealing with this when I was writing some migrations.  I don't believe `drop_if_exists` allows the `concurrently: true` option.  I think if you want the `DROP` to be concurrent you have to use an `execute()`

Reply:
Hmm it seems to be working for me. From my postgres log:

```
2025-07-25 14:01:07.646 PDT [24407] LOG:  execute <unnamed>: DROP INDEX CONCURRENTLY IF EXISTS "flows_account_id_expires_at_gateway_id_index"
```